src/commands/onlydust-import.command.ts

```diff
@@ -19,14 +19,23 @@ export class OnlydustImportCommand extends CommandRunner {
     super();
   }
 
+  // Helper function to check if a username is a GitHub bot
+  isGitHubBot(username: string): boolean {
```

Comment:
```suggestion
  private isGitHubBot(username: string): boolean {
```

Mark the function as private to express this function is internal to this class. Also put it below the main `run()` function to improve the file readability.